src/Farmer/Arm/Network.fs

```diff
@@ -85,6 +87,7 @@ type VirtualNetwork =
                                        else
                                            subnet.AssociatedServiceEndpointPolicies
                                            |> List.map (fun policyId -> {| id = policyId.ArmExpression.Eval() |})
+                                   privateEndpointNetworkPolicies = if subnet.AllowPrivateEndpoints then "Disabled" else "Enabled"
```

Comment:
This is intentional. The `privateEndpointNetworkPolicies ` property prevents the use of private endpoints. However, this seemed confusing to me so I inverted the logic such that, in Farmer, we have `allow_private_Endpoints` rather than 'Disable the thing that prevents private endpoints".
I've moved the inversion to the builder and added a comment to this effect